packages/core/src/integrations/inboundfilters.ts

```diff
@@ -178,10 +178,12 @@ export class InboundFilters implements Integration {
     try {
       if (event.stacktrace) {
         // tslint:disable-next-line:no-unsafe-any
-        return (event as any).stacktrace.frames[0].filename;
+        const frames = (event as any).stacktrace.frames;
```

Comment:
I know we didn't do this before, but can we add more checks here in both cases something like:

```
if (event.stacktrace && event.stacktrace.frames && event.stacktrace.frames.length > 0) 
```

Reply:
That's why it's wrapped inside `try/catch`, so we don't have to do this :)